### ISSUE TITLE
declaring plugins in the JBrowse trackList.json and the Apollo should not disable plugins

### ISSUE THREAD

**Issue by nathandunn** [GMOD/Apollo#2014]
In reference to this:

http://gmod.827538.n3.nabble.com/elastic-search-in-genome-apollo-tp4059909p4060113.html

We either need to remove plugins from the jbrowse trackList.json (to be re-added by Apollo) or just guarantee they do not collide

**Comment by cmdcolin** [comment]
This is potentially something jbrowse should check for on it's side of things. A bug exists for it here https://github.com/GMOD/jbrowse/issues/897

**Comment by nathandunn** [comment]
Yeah, this is a tricky issue.   Ultimately Apollo is reading in the trackList.json and "faking it" when it comes in, so we might be able to fix it for our use-case of users re-using the same directory or config files.  . . .  I'll watch it if you get anywhere on it.   We still have to get to 1.16 if we can.

**Comment by nathandunn** [comment]
This has already been worked around on another bug.

**Comment by nathandunn** [comment]
The other issue was this: https://github.com/GMOD/Apollo/pull/2007/files#diff-2fd45819f7cf7c70bbdc718d5b19faefR465 

But the reason for the fix was this: https://github.com/GMOD/Apollo/pull/2007#issuecomment-456572372
